src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -8,6 +8,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn),649</NoWarn>
     <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
+    <!-- We do not want to block reflection for this assembly -->
+    <BlockReflectionAttribute Condition="'$(TargetGroup)'=='uapaot'">false</BlockReflectionAttribute>
```

Comment:
I'm adding the above line to enable reflection on `System.Private.Xml.dll`. The tests of XmlSerializer need to use reflection to change the serialization mode to test reflection based serialization, see [this line](https://github.com/dotnet/corefx/blob/7575bee96007a5994968fc31d685ae5e6c874ed3/src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs#L28). This change would enable us to test reflection based serialization on UWP.

/cc: @morganbr @weshaggard @danmosemsft